### PR TITLE
2 packages from c-cube/linol at 0.2

### PR DESCRIPTION
**Pull request by c-cube**
This pull-request concerns:
-`linol.0.2`: LSP server library
-`linol-lwt.0.2`: LSP server library (with Lwt for concurrency)

---
* Homepage: https://github.com/c-cube/linol
* Source repo: git+https://github.com/c-cube/linol.git
* Bug tracker: https://github.com/c-cube/linol/issues

---
:camel: Pull-request generated by opam-publish v2.0.0

### CODE REVIEW

**Comment by c-cube** [comment]
lint failed, but `opam lint` succeeded :frowning:

**Comment by mseri** [comment]
The lint here checks a few more things (some will move to opam 2.1 though).
In this case it fails since there is no license field. If you add it to both files I’ll be happy to merge

**Comment by c-cube** [comment]
can you do it here and I'll do it upstream? It's annoying to have to re-release just to appease the linter…

**Comment by mseri** [comment]
Sure. I'll put MIT as mentioned in the readme, ok?

**Comment by c-cube** [comment]
Perfect, did the same upstream.

File: packages/linol-lwt/linol-lwt.0.2/opam [hunk 1]
```diff
@@ -0,0 +1,32 @@
+opam-version: "2.0"
+maintainer: "simon.cruanes.2007@m4x.org"
+homepage: "https://github.com/c-cube/linol"
+synopsis: "LSP server library (with Lwt for concurrency)"
+build: [
```
Comment (mseri) [inline packages/linol-lwt/linol-lwt.0.2/opam:5]:
```suggestion
license: "MIT"
build: [
```

File: packages/linol/linol.0.2/opam [hunk 2]
```diff
@@ -0,0 +1,28 @@
+opam-version: "2.0"
+maintainer: "simon.cruanes.2007@m4x.org"
+homepage: "https://github.com/c-cube/linol"
+synopsis: "LSP server library"
+build: [
```
Comment (mseri) [inline packages/linol/linol.0.2/opam:5]:
```suggestion
license: "MIT"
build: [
```

**Pull request merged**